Doc/library/threading.rst

```diff
@@ -359,6 +359,10 @@ since it is impossible to detect the termination of alien threads.
    If ``None`` (the default), the daemonic property is inherited from the
    current thread.
 
+   *context* is the `contextvars.Context` value to use while running the thread.
+   The default is to inherit the context of the caller of :meth:`~Thread.start`.
```

Comment:
I think passing `None` should by default should mean that the copy of the current context is being used. if empty context is required, users can just pass empty context `Context()`. This would make it behave similar to how it works in asyncio. 

https://docs.python.org/3.13/library/asyncio-task.html#asyncio.create_task

Reply:
Seems good to me to follow what asyncio does here, changed.